data/mods/gen7/moves.ts

```diff
@@ -545,6 +545,24 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		inherit: true,
 		isNonstandard: null,
 	},
+	lunardance: {
+		inherit: true,
+		condition: {
+			duration: 2,
+			onSwitchInPriority: 1,
+			onSwitchIn(target) {
+				if (!target.fainted) {
+					target.heal(target.maxhp);
+					target.setStatus('');
+					for (const moveSlot of target.moveSlots) {
+						moveSlot.pp = moveSlot.maxpp;
+					}
+					this.add('-heal', target, target.getHealth, '[from] move: Healing Wish');
```

Comment:
Probably copied too much from the Healing Wish implementation ;)

Reply:
Indeed, and on the next line too. Whoops!